src/guides/collator_deployment.md

```diff
@@ -0,0 +1,89 @@
+# Introduction
+Collators are members of the network that maintain the parachains they take part in. They run a full node (for both their particular parachain and the relay chain), and they produce the state transition proof for relay chain validators.
+
+There are some requirements that need to be considered prior to becoming a collator candidate including machine, bonding, account, and community requirements.
+For more info regarding collators, reference [collator section](https://wiki.polkadot.network/docs/learn-collator) on Polkadot wiki page.
+
+# prerequisites
+Make sure your instance has at least the minimum recommended harware requirements:
+- CPU: 2 Cores - Intel Xeon E-2386/2388 or Ryzen 9 5950x/5900x
+- Memory: 8GB 
+- Disk: Depending on the chain size - 200GB SSD/nvme
+
+# Launch the Collator 
+
+## Local Deployment
+
+```bash
+./target/release/parachain-template-node --collator \
+--name <collator-name> \
+--base-path /var/parachain/<collator-name> \
+--chain parachain-chainspec-raw.json \
+--force-authoring \
+--rpc-port 10001 \
+--ws-port 10002 \
+--listen-addr /ip4/0.0.0.0/tcp/10003/ws \
+-- \
+--execution wasm \
+--chain /{PATH_TO_RELAYCHAIN_CHAINSPEC} \
+--port 10004 \
+--ws-port 10005
+```
+
+## Kubernetes
+If you wish to run the collator on kubernetes environment, there's already a node helm-chart developed by parity that supports collators.
+
+#### Add Node Helm Chart Repo
+```bash
+helm repo add parity https://paritytech.github.io/helm-charts/
+helm -n NAMESPACE upgrade --install RELEASE_NAME -f FILE_NAME.yml parity/node
+```
+
+#### Install the node helm chart
+Here's an example values file that will be needed to properly deploy the parachain:
+```bash
+node:
```

Comment:
This YAML is from the ArgoCD chart. You should remove the root `node` key.

You can use the minimal YAML from here: [parachain.yaml](https://github.com/paritytech/helm-charts/blob/main/charts/node/examples/local-rococo/parachain.yaml).